test/support/functional_test.rb

```diff
@@ -11,12 +11,25 @@ def roast(*args)
   # Returns an array of strings [stdio_output, stderr_output]
   def in_sandbox(with_workflow: nil)
     # have to save our current working directory before entering sandbox
-    candidate_example_path = File.join(Dir.pwd, "examples", with_workflow.to_s)
+    root_project_path = Dir.pwd
+    project_dot_roast_path = File.join(root_project_path, ".roast")
+    candidate_example_path = File.join(root_project_path, "examples", with_workflow.to_s)
 
     capture_io do
       Dir.mktmpdir do |dir|
         Dir.chdir(dir) do |dir|
-          Dir.mkdir(".roast")
+          if Dir.exist?(project_dot_roast_path) && ENV["RECORD_VCR"]
+            FileUtils.cp_r(project_dot_roast_path, ".roast")
```

Comment:
This loads your local roast initializers when running the test, so we can record the HTTP interactions where necessary.